### PR TITLE
Added router functionality and very basic navigation

### PR DESCRIPTION
**Pull request by codingandcaring**
@NickWilsonDev @rfpoulos

### CODE REVIEW

File: src/Header.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+
+let Header = () => 
+    <div className="header">
+        <Link to='/'>Home</Link>
+        <Link to='/category/'>Category</Link>
```
Comment (rfpoulos) [inline src/Header.js]: I would change this to "set" instead of "category" to keep vocab consistent.

File: src/Header.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+        <Link to='/item'>Individual Item</Link>
```
Comment (rfpoulos) [inline src/Header.js]: "card" instead of "item"

File: src/ItemScreen.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+let ItemScreen = () => 
+    <div>
+        <p>ItemScreen</p>
```
Comment (rfpoulos) [inline src/ItemScreen.js]: "CardScreen"

File: src/App.js [hunk 4]
```diff
@@ -1,21 +1,7 @@
...
-          <h1 className="App-title">Welcome to React</h1>
-        </header>
-        <p className="App-intro">
-          To get started, edit <code>src/App.js</code> and save to reload.
-        </p>
-      </div>
-    );
-  }
-}
+let App = () => <Router className="router"/>
```
Comment (rfpoulos) [inline src/App.js]: What does the router have a class name?

File: src/index.js [hunk 5]
```diff
@@ -4,5 +4,5 @@ import './index.css';
 import App from './App';
 import registerServiceWorker from './registerServiceWorker';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(<App className="App"/>, document.getElementById('root'));
```
Comment (rfpoulos) [inline src/index.js]: why does this need a className?

Comment (codingandcaring) [reply src/index.js]: Done!

File: src/ItemScreen.js (hunk 3 above)
Comment (codingandcaring) [reply src/ItemScreen.js]: Done!

File: src/Header.js (hunk 2 above)
Comment (codingandcaring) [reply src/Header.js]: Done!

File: src/Header.js (hunk 1 above)
Comment (codingandcaring) [reply src/Header.js]: Made change!

File: src/App.js (hunk 4 above)
Comment (codingandcaring) [reply src/App.js]: I was using CSS to maintain width and height and wanted to ensure this remained consistent from the top.  I took off the class name, but may need to add it back later if we start having difficulty maintaining height to 100%.

**Pull request merged**